pkg/usersettings/types.go

```diff
@@ -0,0 +1,20 @@
+package usersettings
+
+type ResourceData struct {
+	Username string
+	UID      string
+	// The resource suffix uses the User uid like "-12341234-1234-1234-1234-123412341234" or "" for kube:admin
+	ResourceSuffix string
+}
+
+func (r *ResourceData) getConfigMapName() string {
```

Comment:
Would be good to add unit tests for these methods.

Reply:
I would like to move this from "name getters / helpers" to the moment when creating the ResourceNames. See comment above.

But for the moment I added tests for the "name getters / helpers" here as well.